### PR TITLE
feat(mailer): Design mail templates to send mails based on specific conditions

### PR DESCRIPTION
**Pull request by printf-twinkle**
### Functionality:
Designing mails to fix readability issues

### Solution:
Dedicated condition based html files to send mails for particular reasons.

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
Complete any functionality involving mails.

### CODE REVIEW

File: functions/model/mailer/tark/generateTemplate.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      htmlContent = htmlContent.replace("$$$", valueArray[2]);
+        htmlContent = htmlContent.replace("$$$$", valueArray[1]);
+        htmlContent = htmlContent.replace("$$$$", valueArray[0]);
+        htmlContent = htmlContent.replace("https://worktez.com/verifyUser/$$$$", "https://worktez.com/verifyUser/"+valueArray[3]+valueArray[0]+valueArray[4]+valueArray[2]);
+    }
+    htmlbody = data.replace("$$$", htmlContent);
+    subjectMessage = mailSubject;
+    message.push(subjectMessage);
+    message.push(htmlbody);
+    console.log("abc", message);
```
Comment (viv9k) [inline functions/model/mailer/tark/generateTemplate.js:129]: Remove this console.log

File: functions/model/mailer/tark/getTemplates.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable object-curly-spacing */
+/* eslint-disable eol-last */
+/* eslint-disable indent */
+/* eslint-disable max-len */
+/** *********************************************************
+ * Copyright (C) 2022
+ * Worktez
+ * Author: Twinkle Chatterjee (ctwinkle2812@gmail.com)
```
Comment (viv9k) [inline functions/model/mailer/tark/getTemplates.js:9]: Please put my name and email here

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: functions/model/mailer/tark/generateTemplate.js (hunk 1 above)
Comment (printf-twinkle) [reply functions/model/mailer/tark/generateTemplate.js:129]: done

File: functions/model/mailer/tark/getTemplates.js (hunk 2 above)
Comment (printf-twinkle) [reply functions/model/mailer/tark/getTemplates.js:9]: done

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**